angr/slicer.py

```diff
@@ -211,16 +211,21 @@ def _backward_handler_stmt(self, stmt, state):
 
         return in_slice
 
+    #def _backward_handler_stmt_Exit(self, stmt, state):
+    #    # HACK: TODO: WARNING: This might not be what we want.
+    #    # You may end up with bigger slices than you expect
+    #    # However, since a conditional exit reflects your ability to continue
+    #    # to follow the slice, all conditional exits must be included
+    #    # (until we find a more clever way to exclude them)
+    #    self._backward_handler_expr(stmt.guard, state)
+    #    return True
+
```

Comment:
Umm why are you commenting out the above lines? I thought we have fixed the problems you encounter without nopping this method out.

Reply:
You have this backwards. I added this myself in the first place because it was needed, then you fixed something else completely different with slicer that made this unnecessary, so I eventually took it out, and everything still works.